hana/defaults.yaml

```diff
@@ -41,3 +41,4 @@ hana:
         remote_instance: '00'
         replication_mode: 'sync'
         operation_mode: 'logreplay'
+        timeout: 100
```

Comment:
@arbulu89  we noticed in our tests that it can take much longer sometimes. Maybe something like 600 would be more appropriate as default?